AaSystem/AaSystemControl/AaUpgrade.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+from AaSystem.LogAndPrint.Log import PrintRedAndLog, PrintAndLog
```

Comment:
why not to use default logging system?
https://docs.python.org/2/library/logging.html

Reply:
No reason, I really should update to use that, currently not really logging anything since I need to make some updates to permit saving the logs (permission issues)